reference/glossary.rst

```diff
@@ -304,220 +304,143 @@ Glossary
         *Work in Progress*
 
     DAC
-        *Work in Progress*
+        Discretionary access control
+        A form of access control where the owner of a resource can grant/revoke permissions to other users.
```

Comment:
```suggestion
        Discretionary access control

        A form of access control where the owner of a resource can grant/revoke permissions to other users.
```

Reply:
In the rendered documentation page, the line breaks you've put here are not being respected and all the text is running together. I think you might need to try putting empty lines between any lines that you want to separate on the page

Reply:
An alternative could be to have any expanded acronym formatted a particular way (e.g. in bold face), and then a colon to separate the description? E.g.
```
**Discretionary Access Control**: A form of...
```
Since we don't have a standard formatting yet, I'll let you decide :)